spec/std/socket/udp_socket_spec.cr

```diff
@@ -2,7 +2,7 @@ require "./spec_helper"
 require "socket"
```

Comment:
The rest of the specs in this directory uses

```crystal
require "./spec_helper"
require "../../support/errno"
```

And that will allow running this file independently

```shell
$ ./bin/crystal spec spec/std/socket/udp_socket_spec.cr 
```

(I have Darwin 18.0.0 still and as the comment stated it requires 18.2.0)

Reply:
Good catch, I've added that include